concent_api/api-integration-force-accept-or-reject-results-test.py

```diff
@@ -101,40 +94,87 @@ def report_computed_task(timestamp = None, subtask_id = None, task_to_compute =
         )
 
 
+def create_task(task_to_compute_timestamp, task_id, compute_task_def_deadline):
+    return task_to_compute(
+        timestamp=task_to_compute_timestamp,
+        compute_task_def=compute_task_def(
+            task_id=task_id,
+            subtask_id=task_id,
+            deadline=compute_task_def_deadline,
+        )
+    )
+
+
+ProtocolConstants = namedtuple("ProtocolConstants",
+                               ["concent_messaging_time",
+                                "subtask_verification_time",
+                                "force_acceptance_time",
+                                "token_expiration_time"])
+
+
+def get_protocol_constants(cluster_url):
+    url = f"{cluster_url}/api/v1/protocol-constants/"
+    resp = requests.get(url)
+    json = resp.json()
+    concent_messaging_time = json['concent_messaging_time']['value']
```

Comment:
It has changed, there will be no `name` and `value`, just the value you can get directly from
`json['concent_messaging_time']` etc.

Reply:
has this change been introduced to master already?

Reply:
Yes.